lib/taskana-core/src/main/java/pro/taskana/common/internal/util/CustomHoliday.java

```diff
@@ -0,0 +1,49 @@
+package pro.taskana.common.internal.util;
+
+import java.util.Objects;
+
+public final class CustomHoliday {
```

Comment:
What do you think of reusing our generic Pair class? Or would that be confusing?

Reply:
First I also thought about it, but I think it's better to be more strict with this object. And in the open source frameworks for similar task I found some examples where the use additional Information at the holidays object - like a description or a name. at this point it will be not a pair anymore. But you are right, the code looks very much the same right now.